README.md

```diff
@@ -1,3 +1,87 @@
-# zs-rediz-rate-limiter
+# rediz-rate-limiter
 
-ZS Library: zs-rediz-rate-limiter
+Redis-based rate limiting.
+
+## Usage
+Rate-limiting is accomplished by creating an instance and calling the `#check`
+method, which will increment the ongoing count for the provided key or, if
+the limit has been reached, reject with a limit-exceeded error.
+
+Check calls with the same key will be tracked together, regardless of which
+process the call was made in, assuming the same redis database is used by
+each process.
+
+The check method requires a completion rate and burst count with each call.
+These can be changed between check calls to the same key, though in practice
+they usually should remain the same.
+
+```js
+const RateLimiter = require('rediz-rate-limiter');
+const RedizClient = require('rediz');
+const pasync = require('pasync');
+
+let client = new RedizClient({ /* redis config */});
+let limiter = new RateLimiter(client);
+
+
+// At most one per 2 seconds.
+let check = () => limiter.check('key', 0.5, 1);
+
+// First check will resolve.
+check()
+	// Second check will reject.
+	.then(() => check().catch(() => {}))
+	// wait 2 seconds.
+	.then(() => pasync.setTimeout(2000))
+	// Third check will resolve.
+	.then(check);
+
+
+// At most two at once, treating two per second as complete.
+// Note that different keys are tracked separately.
+let otherCheck = () => limiter.check('other-key', 2, 2);
+
+// First check will resolve.
+otherCheck()
+	// Second check will resolve.
+	.then(otherCheck)
+	// Third check will reject.
+	.then(() => otherCheck().catch(() => {}))
+	// wait 1/2 second.
+	.then(() => pasync.setTimeout(500))
+	// Fourth check will resolve.
+	.then(check);
+```
+
+## Custom Prefix
+By default, `rediz-rate-limiter` prefixes keys with 'rzrate:' before storing
+them in redis. This prefix can be changed with the RateLimiter constructor.
+Keys with a different prefix will be separated from each other, even if they
+are otherwise the same.
+
+```js
+let asdfLimiter = new RateLimiter(client, 'asdf:');
```

Comment:
I'd recommend always having a `rzrate:` prefix to namespace all rate limiter keys.  The second argument to the constructor should be appended to that.

Additionally, the second argument to the constructor shouldn't need to have a ':'.  That should also automatically be appended.